README.md

```diff
@@ -41,67 +32,6 @@ Requires that you [install Replayable Desktop](https://replayable.io). Replayabl
 
 Also see [the examples folder](https://github.com/replayableio/cli/tree/main/examples).
 
-## Web
-
-### Setup
-
-Nothing! The app exposes the protocol to the system natively via `replayable://`.
-
-### HTML Anchor Tag
-
-```html
-<a href="replayable://replay/create" target="_blank">Create a Replay</a>
-```
-
-### JS Error Handler
-
-```js
-window.onerror = function myErrorHandler() {
-  window.open("replayable://replay/create", "_blank");
-};
-
-setTimeout(() => {
-  throw new Error("Throw makes it go boom!");
-}, 3000);
-```
```

Comment:
This stuff is probably out of scope, but I don't want to remove it until we have a centralized place for it.